File: Beecrowd/Beginner/1000 - 1099/1066 - Even, Odd, Positive and Negative/Python/1066_EvenOddPositiveandNegative.py
even = odd = positive = negative = 0

for i in range(5):
    number = int(input())

    if number % 2 == 0:
        even += 1
    else:
        odd += 1

    if number > 0:
        positive += 1
    elif number < 0:
        negative += 1

print(f"{even} valor(es) par(es)")
print(f"{odd} valor(es) impar(es)")
print(f"{positive} valor(es) positivo(s)")
print(f"{negative} valor(es) negativo(s)")

File: Beecrowd/Beginner/1000 - 1099/1066 - Even, Odd, Positive and Negative/C++/1066_EvenOddPositiveandNegative.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int even = 0, odd = 0, positive = 0, negative = 0;

    for (int i = 0; i < 5; i++) {
        int number;
        cin >> number;

        if (number % 2 == 0) even++;
        else odd++;

        if (number > 0) positive++;
        else if (number < 0) negative++;
    }

    cout << even << " valor(es) par(es)" << endl;
    cout << odd << " valor(es) impar(es)" << endl;
    cout << positive << " valor(es) positivo(s)" << endl;
    cout << negative << " valor(es) negativo(s)" << endl;
    
    return 0;
}